_dashboards/im-dashboards/index.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: default
-title: Index management in Dashboards
+title: Index management
 nav_order: 80
 has_children: true
```

Comment:
Actually, you also need to change the parent name for all children of this page, otherwise they don't show up.